File: 0683/sesión 15  - Master Detail/con Master Detail/SoftInv/SoftInvModel/AlmacenesDTO.cs
namespace SoftInv.Model
{
    public class AlmacenesDTO
    {
        private int? almacenId;
        private string nombre;
        private bool? almacenCentral;

        public AlmacenesDTO()
        {
            this.AlmacenId = null;
            this.Nombre = null;
            this.AlmacenCentral = null;
        }
        public AlmacenesDTO(int almacenId, string nombre, bool almacenCentral)
        {
            this.AlmacenId = almacenId;
            this.Nombre = nombre;
            this.AlmacenCentral = almacenCentral;
        }

        public int? AlmacenId { get => almacenId; set => almacenId = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public bool? AlmacenCentral { get => almacenCentral; set => almacenCentral = value; }
    }
}

File: 0683/sesión 19 - SOAP (2)/Back-End/SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/AlmacenesDTO.java
package pe.edu.pucp.softinv.model;

public class AlmacenesDTO {

    private Integer almacenId;
    private String nombre;
    private Boolean almacen_central;

    public AlmacenesDTO() {
        this.almacenId = null;
        this.nombre = null;
        this.almacen_central = null;
    }
    
    public AlmacenesDTO(Integer almacenId, String nombre, Boolean almacen_central) {
        this.almacenId = almacenId;
        this.nombre = nombre;
        this.almacen_central = almacen_central;
    }

    /**
     * @return the almacenId
     */
    public Integer getAlmacenId() {
        return almacenId;
    }

    /**
     * @param almacenId the almacenId to set
     */
    public void setAlmacenId(Integer almacenId) {
        this.almacenId = almacenId;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the almacen_central
     */
    public Boolean getAlmacen_central() {
        return almacen_central;
    }

    /**
     * @param almacen_central the almacen_central to set
     */
    public void setAlmacen_central(Boolean almacen_central) {
        this.almacen_central = almacen_central;
    }
    
    
}
